### PR TITLE
Drew create events part 2

### PR DESCRIPTION
**Pull request by DrewStableBoi**
Added address info to the front end for creating events, Paul should be the reviewer here to make sure things line up on the back end

### CODE REVIEW

File: src/components/createevents.js [hunk 1]
```diff
@@ -39,28 +46,44 @@ class createEvent extends Component {
...
         is_public_event,
         max_player,
-        creator_id
+        message,
+        locationStreet_one,
+        locationStreet_two,
+        locationCity,
+        locationZip,
+        locationState
+        } = this.state;
```
Comment (Paulyg15) [inline src/components/createevents.js]: @DrewStableBoi Is this supposed to be a this.setState() ?

**Review by Paulyg15 — COMMENTED** [review]
@DrewStableBoi good work, this looks great! I left you a comment on line 74 though. Let me know and then I'll merge it

File: src/components/createevents.js (hunk 1 above)
Comment (DrewStableBoi) [reply src/components/createevents.js]: @Paulyg15 No, I'm destructuring all those fields from state within the context of the method so I don't have to use this.state."" every time. Make sense?

Comment (Paulyg15) [reply src/components/createevents.js]: @DrewStableBoi , Ya you are totally right! I read that backwards lol. Math is hard

Comment (DrewStableBoi) [reply src/components/createevents.js]: BIG MOOD, brother. Haha it's all good!

**Pull request merged**